haystack/components/extractors/named_entity_extractor.py

```diff
@@ -216,6 +216,7 @@ def from_dict(cls, data: Dict[str, Any]) -> "NamedEntityExtractor":
         try:
             init_params = data["init_parameters"]
             init_params["device"] = ComponentDevice.from_dict(init_params["device"])
+            init_params["backend"] = NamedEntityExtractorBackend[init_params["backend"]]
```

Comment:
I'm a bit confused; where is this `init_params` used? it it's never used inside this function

Reply:
OK, nevermind, I understood now what's going on :)